addons/website/views/snippets/s_freegrid.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_freegrid" name="Free grid">
+    <section class="s_freegrid o_cc o_cc1 pt64 pb64">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item o_grid_item_image g-col-lg-1 g-height-8 col-lg-1 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 2 / 1 / 10 / 2; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_03" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 2; grid-area: 1 / 2 / 6 / 5; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_13" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 3; grid-area: 6 / 2 / 11 / 5; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_10" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-8 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 4; grid-area: 2 / 5 / 10 / 8; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_05" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-10 col-lg-3" style="z-index: 5; grid-area: 1 / 8 / 11 / 11; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_14" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-2 g-height-8 col-lg-2 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 6; grid-area: 2 / 11 / 10 / 13; --grid-item-padding-y: 8px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_16" alt=""/>
+                </div>
+                <div class="o_grid_item g-height-3 g-col-lg-7 col-lg-7" style="z-index: 7; grid-area: 12 / 2 / 15 / 9; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px;">
```

Comment:
```suggestion
                <div class="o_grid_item g-height-5 g-col-lg-7 col-lg-7" style="z-index: 7; grid-area: 12 / 2 / 17 / 9; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px;">
```
Looks like there is a sizing issue on tablet:

With current grid sizing:
![Screenshot 2024-08-23 at 16 03 49](https://github.com/user-attachments/assets/9891314c-ce30-47fe-a1c5-6e6c62e5e5cd)
With suggestion: 
![Screenshot 2024-08-23 at 16 04 01](https://github.com/user-attachments/assets/5bcb2ab1-7320-451f-bacb-ceb145854124)